docs/_posts/2015-07-09-readfarm-story-wine.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 红酒的故事
+author: read farm
+categories: news
+tags:
+  - UAV
+  - RS
+  - spectrum
+  - grape
+  - wine
+  - Read Farm
+image: assets/2015/07-readfarm-wine/0.png
+---
+
+葡萄酒产业一直以来是所有涉农产业里产业链最长、附加值最高、生态效益最显著的产业。2015年观耘团队开始深耕酿酒葡萄领域，经过反复迭代技术验证，利用大数据技术帮助种植园大幅提高作物生长一致性；尤其是在病虫害管理上团队积累了丰富经验，建立了针对葡萄种植中数种常见高发、高危病虫害数据的核心算法，技术水平和能力得到行业专家及客户的充分认可。业务覆盖范围逐步由加拿大漫延到北美（Napa Valley）、法国波尔多等全球多处葡萄酒核心产区，目前已服务包含八大名酒庄在内的全球约30余家知名葡萄酒企业。
```

Comment:
The article was too short, can we add more (3-5) paragraphs to it?

A good news post should have serval paragraphs, and as we talked before, it might include:

1. A story behind the company
1. Some options which could inspire others
1. The company gets any prizes.

This story was too short to be a good story.

Reply:
Well noted with thx. We'll upgrade it.